rpi_csdt_community/settings.py

```diff
@@ -346,10 +346,10 @@
 
 USE_CACHE = False
 
-try:
-    from local_settings import *  # noqa: F403
-except:
-    pass
+# try:
```

Comment:
You should not comment out local_settings, it's important that the server still be able to, for example, import real keys and passwords.